Makefile

```diff
@@ -1,6 +1,22 @@
 DEFAULT_GOAL: help
 SHELL := /bin/bash
 
+.PHONY: check-lint
+check-lint:
```

Comment:
Do we need 4 separate make targets? I think we just need one "make lint" (this seems standard in most of our other repos) that does the linting and "make fix" that does all the fixing (c.f. https://github.com/freedomofpress/securedrop-tooling/issues/11).

Reply:
Happy to combine them, done.